src/views/Form/Form.jsx

```diff
@@ -23,61 +24,78 @@ import { updateSection } from 'actions/SectionActions'
 //  3. The section and subsection are known so the section will
 //     display the subsection only.
 class Form extends React.Component {
-  constructor(props) {
-    super(props)
-    this.state = { refreshPending: false }
-  }
-
   componentWillMount() {
     this.defaultRedirect()
   }
 
+  componentDidMount() {
+    this.refreshToken()
```

Comment:
I think this is unnecessary. All api calls will refresh the session, so the only time we need to explicitly call /refresh is if someone clicks the "your session is about to expire" button. At the very least we should try not to make this call when we are also making a /save call, which is going to be the bulk of the time we interact with the page.

Reply:
sounds good, I'll remove it.